RecoHGCal/TICL/python/iterativeTICL_cff.py

```diff
@@ -27,6 +28,13 @@
     ticlCLUE3DHighStepTask
 )
 
+from Configuration.ProcessModifiers.kf_cff import kf
```

Comment:
To be honest I did this because it is also how clue3D was injected. I thought it was mainly due to it being quite flexible but if it is not correct I'll happily change it

Reply:
as discussed, let's keep it like this - thanks